mrb_config.rb

```diff
@@ -113,7 +113,12 @@
     l.library_paths << File.expand_path('soil', bvd)
     l.library_paths.uniq!
 
-    l.libraries << 'glfw'
+    unless Platform.darwin?
+      l.libraries << 'glfw' 
```

Comment:
@IceDragon200 I do not know a little better. Do you mean there is a problem in the use of static files?
I 'm glad, if you fix this on behalf of me 🙏